qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -130,6 +130,9 @@
                 vga = "qxl"
         - check_thaw_unfrozen:
             gagent_check_type = thaw_unfrozen
+        - check_query_chardev:
```

Comment:
suggest not use check_xx as it means check qemu-ga cmd in this cfg, you can use gagent_query_chardev

Reply:
could you pls reply my last comment?